src/containerbuddy/config.go

```diff
@@ -63,6 +68,15 @@ func (s *ServiceConfig) MarkForMaintenance() {
 	s.discoveryService.MarkForMaintenance(s)
 }
 
+func (s *ServiceConfig) Deregister() {
+	s.discoveryService.Deregister(s)
+}
+
+const (
+	// Amount of time to wait before killing the application
+	DEFAULT_STOP_TIMEOUT int = 5
```

Comment:
Golang style is not to all-caps these. Should just be lower-case so that we're not exporting it.